packages/connectivity/src/scp-cf/destination/destination-service-types.ts

```diff
@@ -230,6 +230,12 @@ export type DestinationRetrievalOptions = CachingOptions & {
    * By default, IsolationStrategy.Tenant_User is set.
    */
   isolationStrategy?: IsolationStrategy;
+  /**
+   * The call to `destination/{name}` sometimes also performs a token exchange. Such calls can be flaky due to an unstable token exchange endpoint.
+   * This parameter enables 3 retries on the destination by name calls.
+   * By default, the retry is disabled.
```

Comment:
why is it disabled by default? would not enabled be a good default to have more resilience without requiring user action?

Reply:
hm, I was afraid to add load to the destination service @marikaner what do you think?

Reply:
I like disabled by default. Behavior stays as is unless configured otherwise.

Reply:
> Behavior stays as is unless configured otherwise

v3 is our chance to do breaking changes where we see fit. Sure we need to consider if it is worth it, but in this case I'd say being more resilient by default is added usability. Most users don't change options. Is it good to have resilience for most users? I'd say: yes.